packages/hoppscotch-backend/src/mailer/mailer.module.ts

```diff
@@ -1,27 +1,36 @@
-import { Module } from '@nestjs/common';
 import { MailerModule as NestMailerModule } from '@nestjs-modules/mailer';
 import { HandlebarsAdapter } from '@nestjs-modules/mailer/dist/adapters/handlebars.adapter';
-import { MailerService } from './mailer.service';
+import { Module } from '@nestjs/common';
+import { MAILER_FROM_ADDRESS_UNDEFINED } from 'src/errors';
 import { throwErr } from 'src/utils';
-import {
-  MAILER_FROM_ADDRESS_UNDEFINED,
-  MAILER_SMTP_URL_UNDEFINED,
-} from 'src/errors';
+import { MailerService } from './mailer.service';
 
 @Module({
   imports: [
-    NestMailerModule.forRoot({
-      transport:
-        process.env.MAILER_SMTP_URL ?? throwErr(MAILER_SMTP_URL_UNDEFINED),
-      defaults: {
-        from:
-          process.env.MAILER_ADDRESS_FROM ??
-          throwErr(MAILER_FROM_ADDRESS_UNDEFINED),
-      },
-      template: {
-        dir: __dirname + '/templates',
-        adapter: new HandlebarsAdapter(),
-      },
+    NestMailerModule.forRootAsync({
+      useFactory: () => ({
+        transport: {
+          host: process.env.SMTP_DOMAIN,
+          port: 587,
```

Comment:
hardcoded port?

Reply:
It was actually intended as a quick example, since pull requests are usually implemented by the Hoppscotch team themselves. But I refreshed it again with the current main because there were breaking changes.